aurora/contracts/src/AuroraErc20FastBridge.sol

```diff
@@ -320,36 +326,53 @@ contract AuroraErc20FastBridge is Initializable, UUPSUpgradeable, AccessControlU
         IEvmErc20 token = tokenInfo.auroraTokenAddress;
         require(tokenInfo.isStorageRegistered == true, "The token storage is not registered");
 
-        require(address(token) != address(0), "The token is not registered");
+        require(address(token) != address(0) ||
+                UtilsFastBridge.isStrEqual(transferMessage.transferTokenAccountIdOnNear, auroraEngineAccountIdOnNear),
+                "The token is not registered");
 
         uint256 totalTokenAmount = uint256(transferMessage.transferTokenAmount + transferMessage.feeTokenAmount);
 
-        token.transferFrom(msg.sender, address(this), totalTokenAmount);
+        if (address(token) != address(0)) {
```

Comment:
Let's use the `isNativeTransfer` here

Reply:
fixed: https://github.com/aurora-is-near/fast-bridge-protocol/pull/113/commits/3bcebe661a3a0ccf60e139020d774085a4354940